include/wx/animate.h

```diff
@@ -38,10 +38,7 @@ class WXDLLIMPEXP_CORE wxAnimation : public wxObject
     wxAnimation();
     explicit wxAnimation(const wxString &name, wxAnimationType type = wxANIMATION_TYPE_ANY);
 
-#ifdef wxHAS_MEMBER_DEFAULT
-    wxAnimation(const wxAnimation&) wxMEMBER_DEFAULT;
-    wxAnimation& operator=(const wxAnimation&) wxMEMBER_DEFAULT;
-#endif
+    wxDECLARE_DEFAULT_COPY_CLASS(wxAnimation);
```

Comment:
If member-default is not supported, this can cause a warning:
`warning: extra ‘;’ [-Wpedantic]`

Reply:
Yes, I can trigger it too: `warning: extra ';' after member function definition [-Wextra-semi]`.